overcooked_ai_py/data/layouts/random0.layout

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Remove all the layout copy files

Reply:
One reason for me keeping the original layout names (random0, simple, etc) was because the layout names need to match the names stored in the data for the BC model. I'm using the BC data stored here: https://github.com/HumanCompatibleAI/human_aware_rl/tree/master/human_aware_rl/data/bc_runs Alternatively, I could use the newer data in your human_model_theory repo here: https://github.com/HumanCompatibleAI/human_model_theory/tree/micah_dev/human_model_theory/data/bc_runs

The latter data has the correct layout names (as far as I know), but I've been doing all my benchmarking with the former data so I was reluctant to change. Also, the new data has both "overfitted" and "underfitted" versions for each map.

One option is that I keep using the old data, and keep using the old layout names everywhere except when I actually load the layout, in which case I'll just convert the old names to the new names. This would allow to remove all the layout copy files.

Another option is that I start using the new BC data, and then I can use the new layout names everywhere.

Let me know which you think is the better options. And apologies if I've misunderstood this. Thanks!

Reply:
I think the fastest thing to do is to currently do this:

> One option is that I keep using the old data, and keep using the old layout names everywhere except when I actually load the layout, in which case I'll just convert the old names to the new names. This would allow to remove all the layout copy files.

And try to keep this hack easily removable once the naming issue is fixed. It's quite hacky, but I feel like having duplicates in overcooked_ai is worse.

Reply:
I've now removed the duplicate layouts. I tried a few methods to make my changes as non-hacky as possible. But the best I came come up with was this:
- I added a function to utils that converts the layout names *if* an old name is used (so if a new name is used then this function won't do anything): https://github.com/HumanCompatibleAI/human_ai_coord/blob/c30dc0bb01a9f504915df6703b0c5f72f1ae1d3d/human_aware_rl/utils.py#L138
- I then made a few changes to my own code (pbt_hms and ppo_tom) to make things work. The only change I made to the general code was to behavioural_cloning, here: https://github.com/HumanCompatibleAI/human_ai_coord/blob/c30dc0bb01a9f504915df6703b0c5f72f1ae1d3d/human_aware_rl/imitation/behavioural_cloning.py#L91